README.md

```diff
@@ -75,14 +79,14 @@ impl StatusMessage {
     * `promise_and` -- combinator, allows waiting on several promises simultaneously, before executing the callback;
     * `promise_return` -- treats the result of execution of the promise as the result of the current function.
     
-    Follow [examples/cross-contract-high-level](https://github.com/nearprotocol/near-bindgen/tree/master/examples/cross-contract-high-level)
+    Follow [examples/cross-contract-high-level](https://github.com/nearprotocol/near-sdk/tree/master/examples/cross-contract-high-level)
```

Comment:
Let's search for `github.com/nearprotocol` and update them all :smile:

Reply:
Done